content/pages/volunteers.md

```diff
@@ -23,7 +23,7 @@ committee.
 ## How volunteering works
 
 You can sign up for most positions by filling out [this
-form](http://goo.gl/forms/NsOXea6Yjf5t02Oj1). The only exception is volunteering
+form](https://lists.uwaterloo.ca/mailman/listinfo/wics-volunteers). The only exception is volunteering
```

Comment:
this line should be only 80 characters.

Also can you mention that wics-sys isn't taking volunteers now?